packages/demo/src/App.svelte

```diff
@@ -3,7 +3,7 @@
   import fortmaticModule from '@web3-onboard/fortmatic'
   import gnosisModule from '@web3-onboard/gnosis'
   import injectedModule from '@web3-onboard/injected-wallets'
-  import keepkeyModule from '@web3-onboard/keepkey'
+  // import keepkeyModule from '@web3-onboard/keepkey'
```

Comment:
Is keepkey module not working properly?

Reply:
it wasn't at one point but I think you fixed it in the exterminate bugs card

Reply:
it wasn't but I will add back and test